lib/daru/io/importers/excel.rb

```diff
@@ -45,22 +55,26 @@ class Excel < Base
         #   #    3        4    Franz      nil    Paris      nil
         #   #    4        5   George      5.5     Tome    a,b,c
         #   #    5        6  Fernand      nil      nil      nil
-        def initialize(path, worksheet_id: 0)
+        def initialize(path, sheet: 0, headers: true)
           optional_gem 'spreadsheet', '~> 1.1.1'
```

Comment:
@v0dro - Sure, will document the `#optional_gem` method. Meanwhile, intention of having `optional_gem` run at `#initialize` is same as that of conditionally requiring `excelx` : require only when a particular IO module is used. Sure, `optional_gem` can be tweaked to try requiring dependencies when an Importer is required and raise errors (`LoadError`, etc.) when the importer is called.